rfcs/src/intro.md

```diff
@@ -0,0 +1,53 @@
+# Introduction
+
+Kani is an open-source verification tool that uses automated reasoning to analyze Rust programs. In order to 
+integrate feedback from developers and users on future changes to Kani, we decided to follow a light-weight 
+"RFC" (request for comments) process.
+
+## When to create an RFC
+
+As a rule of thumb you should create an RFC if you are making changes that impact the user experience. New 
+APIs, new features, as well as depreciation, should follow an RFC process. We also ask for substantial architectural 
+changes to be made via RFC.
+
+Bugs and smaller improvements to existing features do not require an RFC. If you are in doubt, feel free to create 
+a [feature request](https://github.com/model-checking/kani/issues/new?assignees=&labels=&template=feature_request.md)
+and discuss the next steps in the new issue.
+
+## The RFC process
+
+This is the overall workflow for the RFC process:
+
+```toml
+    Create RFC ──> Receive Feedback  ──> Accepted?
```

Comment:
You might want to note the `recieve feedback`/`revise` loop.

Also, the `Implement` -> `Revise RFC` feedback edge

Reply:
I added the first. The second I added a note to the item below (I'm worried that the diagram is getting too complicated). Is that OK?